<ui version="4.0" >
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QStarDict Setting</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="settingsTabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDictionaries" >
      <attribute name="title" >
       <string>Dictionaries</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="2" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item rowspan="3" row="0" column="3" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Ordered dictionaries:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="orderedDictsList" />
         </item>
        </layout>
       </item>
       <item rowspan="3" row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Disabled dictionaries:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="disabledDictsList" />
         </item>
        </layout>
       </item>
       <item row="1" column="1" colspan="2" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>0</number>
         </property>
         <item row="1" column="0" >
          <widget class="QToolButton" name="moveLeftButton" >
           <property name="text" >
            <string>Back</string>
           </property>
           <property name="icon" >
            <iconset resource="../resources/qstardict.qrc" >:/icons/back.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QToolButton" name="moveDownButton" >
           <property name="text" >
            <string>Down</string>
           </property>
           <property name="icon" >
            <iconset resource="../resources/qstardict.qrc" >:/icons/down.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QToolButton" name="moveUpButton" >
           <property name="text" >
            <string>up</string>
           </property>
           <property name="icon" >
            <iconset resource="../resources/qstardict.qrc" >:/icons/up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QToolButton" name="moveRightButton" >
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../resources/qstardict.qrc" >:/icons/forward.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPopupWindow" >
      <attribute name="title" >
       <string>Popup window</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="useScanBox" >
           <property name="text" >
            <string>Scan selection</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QCheckBox" name="useScanModifierBox" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="text" >
              <string>Scan only when modifier pressed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="modifierKeyBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <item>
              <property name="text" >
               <string>Shift</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Control</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Alt</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Meta</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/qstardict.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>303</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>410</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>379</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel" >
     <x>410</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useScanBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>useScanModifierBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>92</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel" >
     <x>123</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useScanModifierBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>modifierKeyBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>230</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel" >
     <x>273</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
